database.py

```diff
@@ -9,10 +9,11 @@
 
 class DeclarativeABCMeta(DeclarativeMeta, ABCMeta):
     """
-    Empty class to create a mixin between DeclarativeMeta and ABCMeta
+    Empty class to create a mix in between DeclarativeMeta and ABCMeta
```

Comment:
Could you please rever this change? "mixin" is a specific programming term, and should not be split up. See https://en.wikipedia.org/wiki/Mixin for an explanation of the term :)